kernel/syscall.c

```diff
@@ -3933,26 +3933,37 @@ static long _syscall(void* args_)
 
             BREAK(_return(n, ret));
         }
-        case SYS_myst_is_shared_crt:
+        case SYS_myst_get_fork_mode:
```

Comment:
Could you break this out into a function? It is getting kind of big.

Reply:
done